scripts/Upgrade-PowerShell.ps1

```diff
@@ -83,9 +86,12 @@
 Param(
     [string]$version = "5.1",
     [string]$username,
+    [string]$domain,
```

Comment:
Can you not specify the domain as part of the `$username`, like `DOMAIN\user` or `user@DOMAIN.COM`?

Reply:
Actually I don't know and haven't got domain controller at my disposal to check it. Nevertheless in my opinion separate parameter for domain name is more clear and less obscure way for users to login using their domain accounts.